### PR TITLE
average.c :  Fixed style issues and some changes. 

### PR DESCRIPTION
**Pull request by christianbender**
**Fixes issue:**

* Style issues against the [code style conventions](https://github.com/christianbender/cosmos/tree/master/guides/coding_style/c) of cosmos.  

**Changes:**

* I changed the formatting according to the code style conventions.  
* I changed the data type of the variable ```sum```.  
* I changed the output statment.

### CODE REVIEW

File: code/unclassified/src/average/average.c [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 // Code written by Adeen Shukla (adeen-s)
-#include<stdio.h>
+#include <stdio.h>
 
-int main() {
-	int n, sum = 0, tmp = 0, i;
+int 
+main() 
+{
+	int n, tmp = 0, i;
```
Comment (arnavb) [inline code/unclassified/src/average/average.c]: Please fix the indentation.

File: code/unclassified/src/average/average.c [hunk 2]
```diff
@@ -14,6 +17,7 @@ int main() {
 	   sum += tmp;
```
Comment (arnavb) [inline code/unclassified/src/average/average.c]: Could you move i near this loop?

File: code/unclassified/src/average/average.c (hunk 1 above)
Comment (christianbender) [reply code/unclassified/src/average/average.c]: The indentation is with me correct. I see no problem to the previous one.

**Comment by christianbender** [comment]
@arnavb Done.

File: code/unclassified/src/average/average.c [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
-	int n, sum = 0, tmp = 0, i;
+int 
+main() 
+{
+    int n, tmp = 0, i;
+    double sum = 0.0;
     
     printf("Enter the total number of inputs : ");
     scanf("%d", &n);
 	printf("\nEnter the numbers\n");
```
Comment (arnavb) [inline code/unclassified/src/average/average.c]: I meant the indentation of this `printf`.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_cosmos

**Pull request merged**